src/views/layouts/teamDetails.hbs

```diff
@@ -0,0 +1,238 @@
+                <tr>
+                    <th>Spiele verloren</th>
+                    <td style="padding-left: 20px;">{{team.gamesLost}}</td>
+                </tr>
+                <tr>
+                    <th>Spiele unentschieden</th>
+                    <td style="padding-left: 20px;">{{team.gamesDraw}}</td>
+                </tr>
+                <tr>
+                    <th>Tore Gruppenphase</th>
+                    <td style="padding-left: 20px;"><strong>{{team.goalsGroupStage.[0]}}</strong> - {{team.goalsGroupStage.[1]}}</td>
+                </tr>
+                <tr>
+                    <th>Punkte Gruppenphase</th>
+                    <td style="padding-left: 20px;">{{team.points_Group_Stage}}</td>
+                </tr>
+                <tr>
+                    <th>Tore</th>
+                    <td style="padding-left: 20px;"><strong>{{team.goals.[0]}}</strong> - {{team.goals.[1]}}</td>
+                </tr>
+                <tr>
+                    <th>Punkte Gesamt</th>
+                    <td style="padding-left: 20px;">{{team.points_General}}</td>
+                </tr>
+                <tr>
+                    <th>Sekt gewonnen</th>
+                    <td style="padding-left: 20px;">{{team.sektWon}}</td>
+                </tr>
+                <tr>
+                    <th></th>
+                    <td></td>
+                <tr>
+                    <th>Aktueller Rang in der Gruppe</th>
+                    <td style="padding-left: 20px;">{{groupRank}}</td>
+                </tr>
+                <tr>
+                    <th>Aktueller Rang im Turnier</th>
+                    <td style="padding-left: 20px;">{{tournamentRank}}</td>
+                </tr>
+            </table>
+        </div>
+        <div class="col-md-6">
+            <h2 style="text-decoration: underline;">Spieler</h2>
+            <table class="table table-bordered table-striped" style="width: auto; margin-left: 0;">
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Nummer</th>
+                        <th>Aktionen</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {{#each players}}
+                    <tr>
+                        <td style="padding-left: 20px;">{{this.name}}</td>
+                        <td style="padding-left: 20px;">{{this.number}}</td>
+                        <td style="padding-left: 20px;">
+                            <a href="/player/{{this._id}}" class="btn btn-info">Tore anzeigen</a>
+                        </td>
+                    </tr>
+                    {{else}}
+                    <tr>
+                        <td colspan="3" style="text-align: center;">Keine Spieler vorhanden</td>
+                    </tr>
+                    {{/each}}
+                </tbody>
+            </table>
+            <a href="/player" class="btn btn-primary">Zur Spielerverwaltung</a>
```

Comment:
Maybe preselect the team in the dropdown in Spielerverwaltung - would be a nice detail

Reply:
Specific but ok